pool/app/job/assignment_job/assignment_job.ml

```diff
@@ -0,0 +1,85 @@
+module Queue = Sihl_queue.MariaDb
+include Handler
+
+module Job = struct
+  open CCFun
+
+  let handle ?(ctx : (string * string) list option) (_ : Pool_database.Label.t) =
```

Comment:
Why do we use `?ctx` here, we have a `Pool_database.of_ctx_exn` (or `_opt`) function for this.